run-make-check.sh

```diff
@@ -76,28 +76,26 @@ function run() {
 
     CMAKE_BUILD_OPTS="-DWITH_FIO=ON -DWITH_GTEST_PARALLEL=ON"
 
-    # Are we in the CI ?
+    # The following settings are made for improving ccache efficiency
+    # by removing the entropy generated by the date/time embedded in the build
+    CMAKE_BUILD_OPTS="$CMAKE_BUILD_OPTS -D ENABLE_GIT_VERSION=OFF"
+    export SOURCE_DATE_EPOCH=$(date +%D |date -f- +%s)
+    ccache -o sloppiness=time_macros
```

Comment:
Does the user not using ccache for anything else then ceph ? If user had a particular setup, we'll break it. That's why I let that changes in the JENKINS condition.

Reply:
Today not being able to install ccache is not fatal. So it may worth setting a bash -e to insure that we never reach that code out of ccache presence.